tests/test_sqlite.py

```diff
@@ -27,3 +27,6 @@
 PASSWORD_HASHERS = [
     'django.contrib.auth.hashers.MD5PasswordHasher',
 ]
+
+# Needed for live server tests and cannot easily be added with override_settings
+ALLOWED_HOSTS = ['localhost']
```

Comment:
All projects that use `LiveServerTestCase` need this now? If so, I'm not sure that's very nice, backwards-compatibility and usability wise.